src/main/java/org/broadinstitute/consent/http/service/ResearcherService.java

```diff
@@ -166,12 +166,14 @@ private void validateRequiredFields(Map<String, String> properties) {
         });
     }
 
-    private void validateExistentFields(Map<String, String> properties) {
+    private Map<String, String> validateExistentFields(Map<String, String> properties) {
+        Map<String, String> newProps = new HashMap<>();
         properties.forEach((propertyKey, propertyValue) -> {
-            if (!UserFields.containsValue(propertyKey)) {
-                throw new IllegalArgumentException(propertyKey + " is not a valid property.");
+            if (UserFields.containsValue(propertyKey)) {
+                newProps.put(propertyKey, propertyValue);
             }
```

Comment:
There's no need to throw an exception, we just store the values we know and care about.